Lombiq.NodeJs.Extensions/scripts/run-pnpm-script.js

```diff
@@ -0,0 +1,70 @@
+﻿const child_process = require('child_process');
```

Comment:
Do you find it objectionable, or alien?

Reply:
At first, it was too involved for me - I didn't see the big picture. 

Then, I understood that you're extending Node.js Extensions in a way to provide something equivalent for Vue.js.

In the meantime, I'm getting familiar with the Vue.js paradigms.

And now, I'm trying to evaluate where this approach is feasible, and where it can be improved.